plugin/src/main/java/com/denizenscript/denizen/utilities/BukkitImplDeprecations.java

```diff
@@ -344,6 +344,9 @@ public class BukkitImplDeprecations {
     // Bad candidate for bumping, targets extremely commonly used naming, some of which may be hard to remove (eg stored in flag data).
     public static Warning oldSpigotNames = new FutureWarning("oldSpigotNames", "Several features (particles, entities, etc.) had alternative naming added by Spigot, which are now deprecated in favor of the official Minecraft naming; see relevant feature's meta docs for more information.");
 
+    // Added 2024/11/19
+    public static Warning oldPotionEffectType = new FutureWarning("oldPotionEffectType", "Potion effects now use an 'effect' key for the potion effect's type, see meta docs for more information.");

```

Comment:
why tho

Reply:
As in clearer warning text? or why the change at all?

Reply:
was why the rename at all but I figured it out later and forgot to remove this comment oops